near/omni-prover/omni-prover/src/lib.rs

```diff
@@ -61,35 +55,39 @@ impl OmniProver {
     #[must_use]
     pub fn init() -> Self {
         let mut contract = Self {
-            bridges: near_sdk::collections::UnorderedMap::new(StorageKey::RegisteredBridges)
+            bridges: near_sdk::collections::UnorderedMap::new(StorageKey::RegisteredBridges),
         };
 
         contract.acl_init_super_admin(near_sdk::env::predecessor_account_id());
         contract
     }
 
     #[access_control_any(roles(Role::BridgesManager, Role::DAO))]
-    pub fn set_bridge(&mut self, prover_id: ProverId, bridge_account_id: AccountId) {
-        self.bridges.insert(&prover_id, &bridge_account_id);
+    pub fn set_bridge(&mut self, chain_kind: ChainKind, account_id: AccountId) {
```

Comment:
I don't like that we have a limited number of ChainKind, and if we ever want to support a new chain, we'll have to redeploy the contract.

Reply:
> we'll have to redeploy the contract.
Yeh, it sounds legit. In this case, we should revert it to string.